sdk/iot/Azure.Iot.Hub.Service/.gitignore

```diff
@@ -0,0 +1,3 @@
+launchSettings.json
```

Comment:
launchSettings.json is in the root .gitignore file in the public mono repo. This is another sign we need a rebase on master for this feature branch.